terraform/modules/common/outputs.tf

```diff
@@ -17,21 +17,41 @@ output "jvs_api_service_account_email" {
   value       = google_service_account.api_acc.email
 }
 
+output "jvs_api_service_account_id" {
+  description = "JVS API service account id."
+  value       = google_service_account.api_acc.id
+}
+
 output "jvs_ui_service_account_email" {
   description = "JVS UI service account email."
   value       = google_service_account.ui_acc.email
 }
 
+output "jvs_ui_service_account_id" {
+  description = "JVS UI service account id."
+  value       = google_service_account.ui_acc.id
```

Comment:
let's use name instead of id and rename every where. See https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs/resources/google_service_account_iam#google_service_account_iam_member
<img width="628" alt="Screenshot 2023-09-08 at 2 26 05 PM" src="https://github.com/abcxyz/jvs/assets/51539171/4d27e9e0-327a-46c6-ad7a-9719782498a2">

Reply:
Updated